.travis.yml

```diff
@@ -7,14 +7,22 @@ script:
 matrix:
     include:
         - python: 2.7
-          env: TOXENV=py27
+          env: TOXENV=py27-coverage
```

Comment:
Why do we want coverage reports from all the envs. If we're trying to use coverage as a tool for making sure our tests are any good, I would think that just one env needs coverage? Or is it just easier this way?

Reply:
Normally I would have skipped pypy (because it is slower with coverage), but for paste's test suite this is fine.

We should at least have it for py2/py3 because of using different code paths.

Codecov will use the flags to overlay/show which envs/flags are touching which code, and aggregates them all into total coverage.

Reply:
WFM